src/api/payment-requests/payment-requests.md

```diff
@@ -251,7 +251,8 @@ Payment Activities are created when a Payment Request has been **created**, **pa
   example {
     title 'Create a Payment Request with a Patron Code'
     body ({
-      barcode: '1219210961929460',
+      barcode: '458028560',
+      barcodeHint: 'farmlands',
```

Comment:
The title of this one is with a patron code, which is described as a 16 digit number in our docs. I think we should keep this example using our Centrapay patron codes and not farmlands barcodes.